### PR TITLE
chore: bump local-node version

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
- Bumps the version of local-node to use the latest versions of the network and mirror nodes.
- Reorganizes some tests that were failing due to changes from [HIP 540](https://hips.hedera.com/hip/hip-540)
- Removed redundant steps from the test `SafeHTS -> should be able to transfer tokens and hbars atomically`.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by isavov — DISMISSED** [review]

File: test/hts-precompile/utils.js [hunk 1]
```diff
@@ -579,6 +579,10 @@ class Utils {
     }
   }
 
+  static keyListFromContractAddressAndKey(key, contract) {
```
Comment (quiet-node) [inline test/hts-precompile/utils.js:582]: Was wondering why these changes are here in this PR? Is it related to the local-node being bumped to 2.25.2?

File: test/hts-precompile/utils.js [hunk 2]
```diff
@@ -623,7 +627,7 @@ class Utils {
     if (setSupply) tx.setSupplyKey(keyList);
     if (setWipe) tx.setWipeKey(keyList);
 
-    await (
+    return await (
```
Comment (quiet-node) [inline test/hts-precompile/utils.js:630]: Was wondering why these changes are here in this PR? Is it related to the local-node being bumped to 2.25.2?

File: test/safe-hts-precompile/SafeHTS.js [hunk 3]
```diff
@@ -204,16 +204,6 @@ describe('SafeHTS library Test Suite', function () {
     await utils.associateWithSigner(signer1PrivateKey, fungibleTokenAddress);
     await utils.associateWithSigner(signer1PrivateKey, nonFungibleTokenAddress);
 
-    await safeOperationsContract.safeGrantTokenKycPublic(
```
Comment (quiet-node) [inline test/safe-hts-precompile/SafeHTS.js:207]: Was wondering why these changes are here in this PR? Is it related to the local-node being bumped to 2.25.2?

**Review by quiet-node — APPROVED** [review]
signed off but some Qs

File: test/hts-precompile/utils.js (hunk 2 above)
Comment (Ivo-Yankov) [reply test/hts-precompile/utils.js]: Those changes were left by mistake

File: test/safe-hts-precompile/SafeHTS.js (hunk 3 above)
Comment (Ivo-Yankov) [reply test/safe-hts-precompile/SafeHTS.js:207]: This was done due to changes related to hip 540 that were introduced in the lastet network node images. The current test scenario was not working and needed to be reworked.

**Review by quiet-node — APPROVED** [review]
LGTM
